Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -199,6 +199,9 @@ open class ChatChannelListVC: _ViewController,
     }
         
     open func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        defer {
+            collectionView.deselectItem(at: indexPath, animated: true)
+        }
```

Comment:
Does this fixes this issue: https://github.com/GetStream/stream-chat-swift/issues/1794 ?

Reply:
If yes, can we add this fix to the changelog?

Reply:
It does, yeah! Thanks for spotting it 😄